specification/cognitiveservices/OpenAI.Inference/routes.tsp

```diff
@@ -41,10 +43,38 @@ Gets chat completions for the provided chat messages.
 Completions support a wide variety of tasks and generate text that continues from or "completes"
 provided prompt data.
 """)
-@TypeSpec.Rest.actionSeparator("/")
+@added(ServiceApiVersions.v2023_05_15)
+@actionSeparator("/")
 @action("chat/completions") //@convenientAPI(true)
-op getChatCompletions is Azure.Core.ResourceAction<
+op getChatCompletions is ResourceAction<
   Deployment,
   ChatCompletionsOptions,
   ChatCompletions
 >;
+
+#suppress "@azure-tools/typespec-azure-core/no-rpc-path-params" "because"
```

Comment:
```suggestion
#suppress "@azure-tools/typespec-azure-core/no-rpc-path-params" "Allowed because this is a non-standard status polling operation."
```